OSSM-6170 OSSM 2.5.1, 2.4.7, and 2.3.11 [DOC] Release Notes, Known Issues and Bug Fixes (#73889)
openshift/openshift-docs: modules/ossm-rn-known-issues.adoc

These are the known issues in {SMProductName}:

> * https://issues.redhat.com/browse/OSSM-6267[OSSM-6267] After a data source is configured correctly in the Grafana, a data query returns authentication error. Users are not able to view data in the **Istio service** and **Istio workload** dashboards. Currently, no workaround for exists this issue.

Reviewer: ```suggestion
* https://issues.redhat.com/browse/OSSM-6267[OSSM-6267] After a data source is configured correctly in the Grafana, a data query returns authentication error. Users are not able to view data in the **Istio service** and **Istio workload** dashboards. Currently, no workaround exists for this issue.
```

> * https://issues.redhat.com/browse/OSSM-6267[OSSM-6267] After a data source is configured correctly in the Grafana, a data query returns authentication error. Users are not able to view data in the **Istio service** and **Istio workload** dashboards. There is currently no workaround for this issue.

Reviewer 2: See feedback from @bburt-rh about "there is" in the previous peer review.

These are the known issues in Kiali:

> * https://issues.redhat.com/browse/OSSM-6299[OSSM-6299] In {product-title} 4.15, when you click on the **Node graph** menu option of the traffic graph menu of any node, the node graph is not displayed. Instead, the page is refreshed with the same traffic graph. There is currently no workaround for this issue.

Reviewer 3: This is not quite clear to me. I think it is better: 

```suggestion
* https://issues.redhat.com/browse/OSSM-6299[OSSM-6299] In {product-title} 4.15, when you click on the **Node graph** option of any node context menu within the traffic graph, the node graph is not displayed. Instead, the page is refreshed with the same traffic graph. There is currently no workaround for this issue.
```

This is a screenshot of the page:
![image](https://github.com/openshift/openshift-docs/assets/122779323/7387ddb4-fc9d-43ac-b7ca-24d5da6d4e23)

> * https://issues.redhat.com/browse/OSSM-6299[OSSM-6299] In {product-title} 4.15, when you click the **Node graph** menu option of any node menu within the traffic graph, the node graph is not displayed. Instead, the page is refreshed with the same traffic graph. Currently, no workaround exists for this issue.

Reviewer: This is specifically mentioning 4.15. Should this known issue only be present in the 4.15 branch then (instead of in the 4.12-4.14 branches too)?

> * https://issues.redhat.com/browse/OSSM-6298[OSSM-6298] When you click on an item reference within the {SMPlugin}, such as a workload link related to a specific service, the console sometimes performs multiple redirections before landing on the desired page. If you click the "Back" button in the web browser, the console navigates to a different page of the console instead of the previous page.

Reviewer 4: ```suggestion
* https://issues.redhat.com/browse/OSSM-6298[OSSM-6298] When you click an item reference within the {SMPlugin}, such as a workload link related to a specific service, the console sometimes performs multiple redirections before opening the desired page. If you click *Back* in a web browser, the a different page of the console opens instead of the previous page.
```
* Take a look at https://www.ibm.com/docs/en/ibm-style?topic=elements-ui to review the guidelines for referring to UI elements.

+
Workaround: In the web browser, click *Back* twice to navigate to the previous page.

> * https://issues.redhat.com/browse/OSSM-6290[OSSM-6290] For {product-title} 4.15, the **Project** filter of the **Istio Config** list page does not work correctly. All `istio` items are displayed even if you select a specific project from the dropdown. Currently, no workaround exists for this issue.

Reviewer: Same question about 4.15

> * https://issues.redhat.com/browse/OSSM-6290[OSSM-6290] For {product-title} 4.15, the **Project** filter of the **Istio Config** list page does not work correctly. All `istio` items are displayed even if you select a specific project from the dropdown. There is currently no workaround for this issue.

Reviewer 4: ```suggestion
* https://issues.redhat.com/browse/OSSM-6290[OSSM-6290] For {product-title} 4.15, the **Project** filter of the **Istio Config** list page does not work correctly. All `istio` items are displayed even if you select a specific project from the dropdown. Currently, no workaround exists for this issue.
```
* Per the ISG, avoid "there is" and "there are"
